src/uu/mkdir/src/mkdir.rs

```diff
@@ -120,7 +131,7 @@ fn mkdir(path: &Path, recursive: bool, mode: u16, verbose: bool) -> i32 {
     }
 
     if verbose {
-        show_info!("created directory '{}'", path.display());
+        println!("{}: created directory '{}'", executable!(), path.display());
```

Comment:
Is this change needed? Doesn't `show_info!(...)` do the same thing and in a project standardized way?

Reply:
because show_info sends message to stderr. 
println! send to stdout like gnu mkdir